lxd/instance_post.go

```diff
@@ -95,6 +120,107 @@ func instancePost(d *Daemon, r *http.Request) response.Response {
 	// online (only relevant if "?target=<node>" was given).
 	targetNodeOffline := false
 
+	if clustered && !clusterNotification {
```

Comment:
I have adapted this checking for cluster notifications from the instance creation endpoint. Is this even required here?

Reply:
@tomponline thanks, I appreciate the explanation, this makes it clear for me. Good that you mention it, I was trying to align the logic to what is written for the instance creation, but they both differ in a lot of places. I'll refactor them as you have suggested.

Reply:
After investigating the cluster notification for both instance creation and movement, I have put the placement logic for the movement after the potential redirect of the request so that it is only executed on the target machine. Unlike for the creation, I didn't see any cluster notification user agent in place for the movement.